julia/magic.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, shell):
               end='')
         # Flush, otherwise the Julia startup will keep stdout buffered
         sys.stdout.flush()
-        self.julia = Julia(init_julia=True)
+        self._julia = Julia(init_julia=True)
```

Comment:
Previous `self.julia = ...` was shadowing the method `julia`.  It was fine because the method was actually not called directly in IPython magic system.  But it's handy to not shadow for testing.